bin.js

```diff
@@ -208,9 +208,14 @@ if (result !== null) {
 function runTool (args, Tool, version) {
   const onPort = args['on-port']
 
+  if (!onPort) {
+    console.log('To generate the report press: Ctrl + C')
+  }
```

Comment:
We should not print this if it's `--visualize-only`. If it's `--collect-only` we should change the text slightly, something like "To stop data collection press: Ctrl-C'.